From 805728cabf81ddef6a550b95d08f1f6e00e0cf8b Mon Sep 17 00:00:00 2001
From: Antonio Alvarez Feijoo <antonio.feijoo@suse.com>
Date: Tue, 26 Sep 2023 08:12:23 +0200
Subject: [PATCH] fix(systemd-pcrphase): rename systemd-pcrphase binary to
 systemd-pcrextend

The `systemd-pcrphase` binary has been renamed to `systemd-pcrextend` since
https://github.com/systemd/systemd/commit/32295fa0
---
 modules.d/01systemd-pcrphase/module-setup.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/modules.d/01systemd-pcrphase/module-setup.sh b/modules.d/01systemd-pcrphase/module-setup.sh
index 3dbb4974ef..b9817e6405 100755
--- a/modules.d/01systemd-pcrphase/module-setup.sh
+++ b/modules.d/01systemd-pcrphase/module-setup.sh
@@ -6,7 +6,7 @@
 check() {
 
     # If the binary(s) requirements are not fulfilled the module can't be installed.
-    require_binaries "$systemdutildir"/systemd-pcrphase || return 1
+    require_binaries "$systemdutildir"/systemd-pcrextend || return 1
 
     # Return 255 to only include the module, if another module requires it.
     return 255
@@ -27,7 +27,7 @@ depends() {
 install() {
 
     inst_multiple -o \
-        "$systemdutildir"/systemd-pcrphase \
+        "$systemdutildir"/systemd-pcrextend \
         "$systemdsystemunitdir"/systemd-pcrphase.service \
         "$systemdsystemunitdir/systemd-pcrphase.service.d/*.conf" \
         "$systemdsystemunitdir"/systemd-pcrphase-sysinit.service \
